packages/deployments/contracts/src/cli/init/config/testnet/production.ts

```diff
@@ -94,6 +94,46 @@ export const TESTNET_PRODUCTION_INIT_CONFIG: InitConfig = {
         },
       },
     },
+    {
+      name: "nextAlUSD",
+      canonical: {
+        domain: "1735353714",
+        address: "0x990162AFaA06f591c03DD36ECBDa24A8A80F2D0c",
+        decimals: 18,
+      },
+      representations: {
+        /// ARBITRUM-GOERLI
+        "1734439522": {
+          local: "0x2931b84D59f6f570EEBDAaaE182867C5E5D58075",
+          adopted: "0x2931b84D59f6f570EEBDAaaE182867C5E5D58075",
+        },
+        /// OPTIMISM-GOERLI
+        "1735356532": {
+          local: "0x6e56Cb72f4197b8155Cf8e1efaEFB682BC9ae5d1",
+          adopted: "0x6e56Cb72f4197b8155Cf8e1efaEFB682BC9ae5d1",
+        },
+      },
+    },
+    {
+      name: "nextAlETH",
```

Comment:
should match the canonical symbol
```suggestion
      name: "alETH",
```